pkg/build/build.go

```diff
@@ -1645,6 +1654,13 @@ func (b *Build) BuildPackage(ctx context.Context) error {
 		if err := b.Runner.StartPod(ctx, cfg); err != nil {
 			return fmt.Errorf("unable to start pod: %w", err)
 		}
+		if !b.DebugRunner {
```

Comment:
```suggestion
		if !b.DebugRunner && !b.IsBuildLess() {
```

Reply:
this conditional is already part of the outer:

```go
if !b.IsBuildLess() {
```